libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/EmbeddedMetricHelper.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.di.ipv.core.library.helpers;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import software.amazon.cloudwatchlogs.emf.logger.MetricsLogger;
+import software.amazon.cloudwatchlogs.emf.model.DimensionSet;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+import java.util.Map;
+
+import static uk.gov.di.ipv.core.library.helpers.LogHelper.GOVUK_SIGNIN_JOURNEY_ID_DEFAULT_VALUE;
+import static uk.gov.di.ipv.core.library.helpers.LogHelper.LogField.LOG_GOVUK_SIGNIN_JOURNEY_ID;
+
+@ExcludeFromGeneratedCoverageReport
+public class EmbeddedMetricHelper {
+    private static final MetricsLogger METRICS_LOGGER = getMetricsLogger();
+    private static final String CORE_BACK_EMBEDDED_METRICS_NAMESPACE = "CoreBackEmbeddedMetrics";
+
+    @Getter
+    @AllArgsConstructor
+    @ExcludeFromGeneratedCoverageReport
+    public enum Metric {
+        CRI_REDIRECT("criRedirect"),
+        CRI_RETURN("criReturn");
+
+        private final String name;
+    }
+
+    @Getter
+    @AllArgsConstructor
+    @ExcludeFromGeneratedCoverageReport
+    public enum Dimension {
+        CRI("cri");
+
+        private final String name;
+    }
+
+    public static void createMetrics(
```

Comment:
A couple of minor points:
- I'm not sure 'create' is the right verb here - the AWS terminology is 'put', but perhaps something like 'record' or 'log' would be clearest?
- Does it need to be plural? At the moment we only record one at a time, so feels a bit YAGNI